### PR TITLE
Replaced all JUnit4 Assertions(org.junit.Assert.*) with Junit5 Jupiter Assertions(org.junit.jupiter.api.Assertions).

### PR DESCRIPTION
**Pull request by ps-19**
## Type of change

 - [`x` ] Bug fix (non-breaking change which fixes an issue)
 - [`x` ] Feature (non-breaking change which adds functionality)
 - [`x` ] Breaking change (fix or feature that would cause existing functionality to change
 - [`x` ] Chore (non-breaking change which doesn't affect codebase;
   test, version modification, documentation, etc.)

## Checklist
 - [`x` ] Code contributed by me aligns with current project license: [Apache 2.0](https://www.apache.org/licenses/LICENSE-2.0)
 - [`x` ] I Added [CHANGELOG](../CHANGELOG.md) entry regarding this change
 - [`x` ] I have implemented unit tests to cover my changes
 - [`x` ] I have added/updated the [javadocs](https://www.javadoc.io/doc/io.fabric8/kubernetes-client/latest/index.html) and other [documentation](https://github.com/fabric8io/kubernetes-client/blob/master/doc/CHEATSHEET.md) accordingly
 - [`x` ] No new bugs, code smells, etc. in [SonarCloud](https://sonarcloud.io/dashboard?id=fabric8io_kubernetes-client) report
 - [ ] I tested my code in Kubernetes
 - [ ] I tested my code in OpenShift

Screenshot 2021-06-04 at 11-29-38 fabric8io kubernetes-client

### CODE REVIEW

**Review by oscerd — APPROVED** [review]

**Comment by ps-19** [comment]
Please help,
First time contributing so may have please review and tell my error.

**Comment by oscerd** [comment]
There are compilation errors on BaseOperationTest

File: kubernetes-client/src/test/java/io/fabric8/kubernetes/client/dsl/base/BaseOperationTest.java [hunk 1]
```diff
@@ -21,8 +21,12 @@
 import static org.hamcrest.CoreMatchers.is;
 import static org.hamcrest.CoreMatchers.notNullValue;
 import static org.hamcrest.MatcherAssert.assertThat;
+<<<<<<< HEAD
+import static org.junit.jupiter.api.Assertions.assertEquals;
+=======
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertTrue;
+>>>>>>> d810aa1da99ff5cdb81cabea4f5a6dcf55ff0218
```
Comment (oscerd) [inline kubernetes-client/src/test/java/io/fabric8/kubernetes/client/dsl/base/BaseOperationTest.java:29]: This needs to be fixed, probably you forced a merge while rebasing.

**Review by oscerd — CHANGES_REQUESTED** [review]

File: kubernetes-client/src/test/java/io/fabric8/kubernetes/client/dsl/base/BaseOperationTest.java (hunk 1 above)
Comment (ps-19) [reply kubernetes-client/src/test/java/io/fabric8/kubernetes/client/dsl/base/BaseOperationTest.java]: Thank you sir.

**Pull request closed without merging**